### PR TITLE
10 new regions

### PR DESCRIPTION
**Pull request by jim-minter**

### CODE REVIEW

File: .pipelines/prod-release.yml [hunk 1]
```diff
@@ -65,7 +70,10 @@ stages:
     parameters:
       environment: RP-Prod-EuropeSector
       locations:
+      - canadacentral
+      - canadaeast
```
Comment (asalkeld) [inline .pipelines/prod-release.yml:74]: You don't like canada? Trying to move it to europe :smile:

Comment (m1kola) [reply .pipelines/prod-release.yml:74]: We will take it

Comment (mjudeikis) [reply .pipelines/prod-release.yml:74]: Always wanted to visit Canada. Ryanair, here I come.

**Comment by jim-minter** [comment]
@JackQuincy or @Zhuoli please can I have a review on this?

File: .pipelines/prod-release.yml (hunk 1 above)
Comment (JackQuincy) [reply .pipelines/prod-release.yml:74]: In v3.11 we have Canada in low traffic and Japan in row. But this should be fine

**Pull request merged**